admin/doc-requirements.txt

```diff
@@ -8,9 +8,9 @@ pyyaml >= 5.1.2
 Cython
 pcpp
 prettytable
-sphinx-autodoc-typehints == 1.18.3
+sphinx-autodoc-typehints ~= 1.18.3
 sphinx-prompt
-sphinx_rtd_theme == 1.0.0
+sphinx_rtd_theme ~= 1.0.0
```

Comment:
Why are the changes here also required?

Reply:
i think it isn't needed! i removed them now. let's see if the build passes